### PR TITLE
Youtube thumbnail uploader

### PR DESCRIPTION
**Pull request by holiber**
image

### CODE REVIEW

**Comment by gettinToasty** [comment]
more of an implementation question but why not just use our existing media gallery input here?

**Comment by holiber** [comment]
The media-gallery-input has a different scenario of usage. It uploads an image to our server, however for this task I need to upload the image directly to YT. I  wish I could use only styles from the media-gallery-input

File: app/services/utils.ts [hunk 1]
```diff
@@ -221,4 +221,14 @@ export default class Utils {
...
   }
+
+  static getReadableFileSizeString(fileSizeInBytes: number): string {
+    let i = -1;
+    const byteUnits = [' kB', ' MB', ' GB', ' TB', 'PB', 'EB', 'ZB', 'YB'];
+    do {
+      fileSizeInBytes = fileSizeInBytes / 1024;
+      i++;
+    } while (fileSizeInBytes > 1024);
+    return Math.max(fileSizeInBytes, 0.1).toFixed(1) + byteUnits[i];
```
Comment (iKBAHT) [inline app/services/utils.ts:232]: What if `i >= byteUnits.length`?

Comment (gettinToasty) [reply app/services/utils.ts:232]: while this is technically unclamped i think the probability of someone making and successfully uploading a 1025+YB file is extremely improbable, and the only place we use it has a max filesize of 2MB

File: app/services/platforms/youtube.ts [hunk 2]
```diff
@@ -656,3 +694,9 @@ export class YoutubeService extends BasePlatformService<IYoutubeServiceState>
     this.state.categories = categories;
   }
 }
+
+export class YoutubeView extends ViewHandler<IYoutubeServiceState> {
+  getThumbnailUploadUrl(videoId: string) {
+    return;
+  }
+}
```
Comment (gettinToasty) [inline app/services/platforms/youtube.ts:702]: is this unused?

**Review by gettinToasty — CHANGES_REQUESTED** [review]
looks like there is some unused code here but otherwise seems good

**Review by gettinToasty — APPROVED** [review]

**Pull request merged**